### PR TITLE
Fix The Atlantic & Telegraaf.nl/Ad.nl/Ed.nl & Bloomberg & add DeMorgen.be

### PR DESCRIPTION
**Pull request by magnolia1234**
Fix The Atlantic (cookie)
Fixes Chrome issue: https://github.com/iamadamdev/bypass-paywalls-chrome/issues/394

Fix Telegraaf.nl (timing issue)
Fix for persistent refreshing of paywall-banner.

Fix Ad.nl/Ed.nl (cookie)
Fix for new cookie-scheme (drop specific cookie).

Fix Bloomberg
Fixes issue https://github.com/iamadamdev/bypass-paywalls-chrome/issues/399

Add DeMorgen.be
Request #387

Simplify specific cookie drop/hold
Search for specific cookiename in all available cookie-(sub)domains.

### CODE REVIEW

File: contentScript.js [hunk 1]
```diff
@@ -114,16 +114,21 @@ if (window.location.href.indexOf("bloombergquint.com") !== -1) {
 
 if (window.location.href.indexOf("bloomberg.com") !== -1) {
     document.addEventListener('DOMContentLoaded', () => {
+		const fence = document.querySelector('.fence-body');
+		if (fence)
```
Comment (cimke) [inline contentScript.js]: To maintain readability you should use one of these examples:

```suggestion
		if (fence) fence.classList.remove('fence-body');
```

or

```suggestion
		if (fence) {
		    fence.classList.remove('fence-body');
		}
```

**Comment by magnolia1234** [comment]
Mr Js-lint again ...
I suggest you make a restyling-PR after a new release and see if Adam approves/cares.

**Comment by cimke** [comment]
**Don't take reviews personally**, it's just code. 
It's easier to fix this now instead of creating separate MR, however, thank you for the suggestion.

**Comment by magnolia1234** [comment]
I would focus on functional improvements instead of 'styling preference'.
PS a single restyling (actually just before a new release) would lessen merge conflicts.

**Comment by cimke** [comment]
I agree with you about pre-release MR.

**Comment by iamadamdev** [comment]
We should avoid any styling changes in the future. I know it's a bit of a mess right now though.

Do you guys have any JS linter you prefer to use?

**Comment by magnolia1234** [comment]
Pure functional commits in this PR.
In my Chrome PR were also updates to old contentScript code, but that is merged already.
I agree to not restyle every commit in favor of oversight in changes.

**Pull request merged**